lib/utils.js

```diff
@@ -87,20 +87,23 @@ var Utils = module.exports = {
     var timeZone = null;
     return SqlString.formatNamedParameters(sql, parameters, timeZone, dialect);
   },
-  cloneDeep: function(obj, fn) {
+  cloneDeep: function(obj) {
+    obj = obj || {};
     return _.cloneDeepWith(obj, function (elem) {
-      // Do not try to customize cloning of plain objects and strings
+      // Do not try to customize cloning of arrays or POJOs
       if (Array.isArray(elem) || _.isPlainObject(elem)) {
         return undefined;
       }
+
+      // Don't clone stuff that's an object, but not a plain one - fx example sequelize models and instances
+      if (typeof elem === 'object' && !_.isPlainObject(elem)) {
```

Comment:
Sorry, on 2nd thoughts, I'm not sure. The 2nd check (line 99) is to copy by reference any objects which are *not* plain objects, which is a different condition from `if (Array.isArray(elem) || _.isPlainObject(elem)) ...` which is for plain objects only.